components/prism-odin.js

```diff
@@ -0,0 +1,61 @@
+(function (Prism) {
+	var escapes = /\\(?:["'\\abefnrtv]|0[0-7]{2}|U[\dA-Fa-f]{6}|u[\dA-Fa-f]{4}|x[\dA-Fa-f]{2})/.source;
+
+	function withEscapes(pattern, flags) {
+		return RegExp(pattern.replace(/<escapes>/g, escapes), flags);
+	}
+
+	Prism.languages.odin = {
+		/**
+		 * The current implementation of multiline comments doesn't support nesting.
+		 */
+		'comment': [
+			{
+				pattern: /\/\*[\s\S]*?(?:\*\/|$)/,
+				greedy: true
+			},
+			{
+				pattern: /#![^\n\r]*/,
```

Comment:
Hash bangs are only allowed as the start of the file, right?

```suggestion
				pattern: /^#!.*/,
```

Reply:
I have now double-checked the work of the compiler. `#!` works exactly the same as `//`. This is implicitly [stated](https://odin-lang.org/docs/spec/#comments) in the [outdated] specification.

Reply:
So no, `#!` can be placed anywhere.